server/sonar-server/src/test/java/org/sonar/server/organization/ws/RemoveMemberActionTest.java

```diff
@@ -324,12 +338,14 @@ private TestResponse call(@Nullable String organizationKey, @Nullable String log
     return request.execute();
   }
 
-  private void assertNotAMember(String organizationUuid, int userId) {
-    assertThat(dbClient.organizationMemberDao().select(dbSession, organizationUuid, userId)).isNotPresent();
+  private void assertNotAMember(String organizationUuid, UserDto user) {
+    assertThat(dbClient.organizationMemberDao().select(dbSession, organizationUuid, user.getId())).isNotPresent();
+    assertThat(userIndex.getNullableByLogin(user.getLogin()).organizationUuids()).doesNotContain(organizationUuid);
```

Comment:
As userIndex.getNullableByLogin will be removed someday, it would be best to use es search, for instance : 
    assertThat(
      esTester.client()
        .prepareSearch(INDEX_TYPE_USER)
        .setQuery(termQuery(FIELD_ORGANIZATIONS, nameQuery))
        .get()
        .getHits()
        .getTotalHits()).isEqualTo(0);